config/routes.rb

```diff
@@ -77,7 +77,7 @@ def not_implemented
             constraints: ToolMountPoint.for(:advice_plans)
 
       mount DecisionTrees::Engine => '/tools/:tool_id',
-            constraints: ToolMountPoint.for(:decision_trees)
+            constraints: { tool_id: %r{health-check-questions|workplace-pension-advice-tool} }
```

Comment:
The alternate url/tool_id implementation perfectly accommodates for this. Once the welsh translation are in they will be added as tool_ids. 

The current implementation on the live site does not have locale translations